loadgen/test_settings_internal.cc

```diff
@@ -36,7 +36,10 @@ TestSettingsInternal::TestSettingsInternal(
       sample_index_rng_seed(requested.sample_index_rng_seed),
       schedule_rng_seed(requested.schedule_rng_seed),
       accuracy_log_rng_seed(requested.accuracy_log_rng_seed),
-      accuracy_log_probability(requested.accuracy_log_probability) {
+      accuracy_log_probability(requested.accuracy_log_probability),
+      issue_unique(requested.issue_unique),
+      issue_same(requested.issue_same),
```

Comment:
Can you log an error somewhere if issue_unique and issue_same are both true?

As currently implemented, I think which one is effective depends on the active scenario.